src/App.js

```diff
@@ -1,11 +1,15 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom"
 import { Home } from "./pages/home/home"
+import { ProductsPage } from "./pages/productsPage/products";
 import Mockman from "mockman-js"
 function App() {
   return (
     <div className="App">
-      <Home />
+      <Routes>
+        <Route exact path = "/" element = {<Home/>} />
+        <Route path = "products" element = {<ProductsPage/>} />
+      </Routes>
```

Comment:
Can create separate file for routes and then import it in App

Reply:
Ok, thank you for suggestion!